Library/Formula/voikko-fi.rb

```diff
@@ -0,0 +1,23 @@
+class VoikkoFi < Formula
+  desc "Finnish dictionary for Voikko speller"
+  homepage "http://voikko.puimula.org"
+  url "http://www.puimula.org/voikko-sources/voikko-fi/voikko-fi-2.0.tar.gz"
+  sha256 "02f7595dd7e3cee188184417d6a7365f9dc653b020913f5ad75d1f14b548fafd"
+
+  depends_on "python3" => :build
+  depends_on "foma" => :build
+  depends_on "libvoikko"
+
+  def install
+    system "make", "vvfst"
+    system "make", "vvfst-install", "DESTDIR=#{prefix}/lib/voikko"
+    lib.install_symlink "voikko"
+  end
+
+  test do
+    require "open3"
+    Open3.popen3("#{HOMEBREW_PREFIX}/bin/voikkospell -m") do |stdin, stdout, _|
```

Comment:
voikko-fi installs only the dictionaries, to test them you need the voikko binary. This depends on libvoikko package being linked, is there a way to refer directly to its install directory?